extensions/README.md

```diff
@@ -28,16 +28,24 @@ Check out these repositories for implementations of the Gardener Extension contr
 
 ### DNS Provider
 
+#### For `DNSRecord` resources (mandatory for Gardener related DNS records)
+
 - [Alicloud](https://github.com/gardener/gardener-extension-provider-alicloud)
 - [AWS](https://github.com/gardener/gardener-extension-provider-aws)
 - [Azure](https://github.com/gardener/gardener-extension-provider-azure)
 - [CloudFlare](https://github.com/schrodit/gardener-extension-provider-dns-cloudflare)
 - [GCP](https://github.com/gardener/gardener-extension-provider-gcp)
 - [OpenStack](https://github.com/gardener/gardener-extension-provider-openstack)
-- [External DNS Management](https://github.com/gardener/external-dns-management)
-  - Alicoud DNS, AWS Route53, Azure DNS, Cloudflare DNS, Google CloudDNS, OpenStack Designate
-  - legacy `DNSProvider/DNSEntry` integration before the [`DNSRecord` API](https://github.com/gardener/gardener/blob/master/docs/deployment/feature_gates.md) was introduced
-  - used for preparing `DNSProviders` for the [Shoot DNS Service](https://github.com/gardener/gardener-extension-shoot-dns-service) if the [`DisableDNSProviderManagement` feature gate](https://github.com/gardener/gardener/blob/master/docs/deployment/feature_gates.md) is not enabled
+
+#### For `DNSEntry` resources
+
+`DNSProvider/DNSEntry/DNSOwner` resources are managed by the [External DNS Management](https://github.com/gardener/external-dns-management)
+extension. It supports providers like Alicoud DNS, AWS Route53, Azure DNS, Cloudflare DNS, Google CloudDNS, OpenStack Designate, Netlify, InfoBlox.
+These resources are only used by 
+- the [Shoot DNS Service](https://github.com/gardener/gardener-extension-shoot-dns-service)
+  to provide managed DNS records
+- the [Shoot Cert Service](https://github.com/gardener/gardener-extension-shoot-cert-service) to create temporary
+   DNS records for DNS challenges to request TLS certificates.
```

Comment:
I think we should drop this entire section from here since this doc is supposed to only enlist native extensions. Hence, only `shoot-{dns,cert}-service` should be present, but `external-dns-management` or `DNSEntry`s is now considered an implementation detail and no first-class citizen in the extensions space anymore.

Reply:
I think it is too early to remove the `DNSEntry` section. It is complicated enough for our users to understand, that there are two different ways to create DNS records. Can't we keep that until the `dns-external` extension is finally gone and the shoot-dns-service covers this aspect?
I suggest to remove it in a follow-up PR needed for the cleanup work for the `DNSProvider` registration.

Reply:
OK, let's do it like this then and clean this up eventually.